identity-and-access-management/iam/reference-content/reproduce-roles-project-api-keys.mdx

```diff
@@ -0,0 +1,66 @@
+---
+meta:
+  title: Reproducing roles and project-scoped API keys with IAM
+  description: This page explains how to generate access similar to previous roles and project-scoped API keys that existed before IAM.
+content:
+  h1: Reproducing roles and project-scoped API keys with IAM
+  paragraph: This page explains how to generate access similar to previous roles and project-scoped API keys that existed before IAM.
+tags: iam
+categories: 
+  - iam
+  - console
+---
+
+[Identity and Access Management](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/) provides a set of concepts and features to easily manage access to your Scaleway Organization.
+
+If you created your Organization before December 5 2022, you may be used to using roles and Project-scoped API keys to grant rights:
+Each user could have a role (Administrator, Billing Administrator or Editor) which gave specific permissions
+In each Project, API keys could be created, which gave full access to all resources within this Project
+
+We strongly recommend that you apply the privilege-of-least principle when setting up access in the Organization: all users and applications should be able to perform the actions they need, but not more. This principle is best achieved by using all available IAM features to define granular permissions for users and API keys. However, to make the migration from the “old” system to the new one easier, on this page we explain how you can reproduce the former system (roles and Project-scoped API keys) under IAM.
+
+
+## How can I reproduce the deprecated roles of Administrator, Billing Administrator and Editor?
+
+When your Organization was migrated to IAM, three [groups](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#group) were created, with three [policies](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#policy) attached to them.
+
+<Lightbox src="scaleway-iam-roles-change.webp" alt="" />
+
+You can keep adding users to these groups if you want to preserve the “old” system of roles and their corresponding rights. Make sure that you therefore do not delete the groups that were created during the migration process.
+
+<Lightbox src="scaleway-iam-roles-change.webp" alt="" />
+
+If you previously deleted these groups from your Organization, you can recreate them by following these steps:
+
+1. [Create a group](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-group/) from the Groups tab, preferably with a name related to the corresponding role (e.g. “Administrators”, “Editors” or “Billing Administrators”)
+
+2. [Create a policy](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-policy/) from the Policies tab, attached to the group you have just created and with the appropriate permission sets:
+
+| Former role | Permission to add and Scope |
```

Comment:
This table needs to be fixed, not displaying correctly. Replace with the following code.

| Former role           	| Necessary Permission Set                              	| Scope for the Permission Set                         	|
|-----------------------	|-------------------------------------------------------	|------------------------------------------------------	|
| Administrator         	| `OrganizationManager` <br><br>`AllProductsFullAccess` 	| Organization <br><br>All and current future Projects 	|
| Editor                	| `AllProductsFullAccess`                               	| All current and future Projects                      	|
| Billing Administrator 	| `BillingManager` <br><br>`OrganizationReadOnly`       	| Organization <br><br>Organization                    	|